plugins/colordialog/dialogs/colordialog.js

```diff
@@ -22,6 +22,14 @@ CKEDITOR.dialog.add( 'colordialog', function( editor ) {
 		html: '&nbsp;'
```

Comment:
Change allows on writing unit test for this bug-fix.

Generally writing test for it was very tricky. Problem appeared when `ok` button is pressed then method `clearHighlight()` is called. What throw an error because unit test doesn't highlight colour in dialog as user does.
So I just add if-statement which check if focus is defined and also rearrange variable declaration to pass JShint.